packages/block-editor/src/components/global-styles/dimensions-panel.js

```diff
@@ -208,7 +208,7 @@ export default function DimensionsPanel( {
 	includeLayoutControls = false,
 } ) {
 	const decodeValue = ( rawValue ) => {
-		if ( typeof rawValue === 'object' ) {
+		if ( rawValue && typeof rawValue === 'object' ) {
```

Comment:
```suggestion
		if ( rawValue !== null && typeof rawValue === 'object' ) {
```

While the problem can be fixed as is, I think it is more rigorous to explicitly check for `null` or not. This implementation is similar to the [isObject() function](https://github.com/WordPress/gutenberg/blob/95336bf5b79af116cb63017fe59c31956422852b/packages/blocks/src/api/registration.js#L134-L136).

Reply:
Thanks for the feedback! Usage isn't very consistent across Gutenberg, and I see a few other places where similar functions use a truthy check, e.g. in [another isObject function](https://github.com/WordPress/gutenberg/blob/52c9056947cec4c7196d25efdc9a18eba0e6e10b/packages/block-library/src/utils/interactivity/directives.js#L18) and recent `_.isEmpty` removals as in #51098 and https://github.com/WordPress/gutenberg/pull/51096.

I have a slight preference for a truthy check over `!== null` because, though it's less explicit, it seems to me a slightly more complete check and better matches to lodash's non-strict `== null` check in [isEmpty](https://github.com/lodash/lodash/blob/2da024c3b4f9947a48517639de7560457cd4ec6c/isEmpty.js#LL45C12-L45C12).

Let me know if you think we're missing anything by not including the strict check, and we can always open up a follow-up, though!